apps/ui/src/views/Editor.vue

```diff
@@ -319,6 +329,7 @@ export default defineComponent({
         />
         <UiLinkPreview :key="proposalKey || ''" :url="proposal.discussion" />
       </div>
+      <EditorVote v-model="proposal" :error="formErrors.choices" />
```

Comment:
`starknet` and `evm` do not support other voting types so we shouldn't show this field for those.

Reply:
Added  a new helper inside networks helper `isVotingTypeSupported` to handle that